src/port1.0/portconfigure.tcl

```diff
@@ -429,10 +429,20 @@ proc portconfigure::configure_get_universal_archflags {} {
     global configure.universal_archs
     set flags ""
     foreach arch ${configure.universal_archs} {
+        set option ""
+        # passthrough custom options
+        if {[string match "-*" $arch]} {
+            set option " $arch "
+        } elseif {[string match "bitcode" $arch]} {
```

Comment:
That's the same as `$arch eq "bitcode"`, which we prefer because it's much simpler to read.